frontend/components/events/ShareModal.tsx

```
@@ -108,25 +107,33 @@ ${eventURL}`;
                   </div>
+                  <p
+                    id="url"
+                    onClick={highlightURL}
+                    className="mt-4 cursor-pointer hover:bg-gray-50 p-2 rounded break-all"
+                  >
+                    {eventURL}
+                  </p>
+                  <BlackHighlightButton className={`w-full mt-2`} onClick={copyURL} disabled={copied}>
+                    {copied ? (
+                      <span className="flex items-center justify-center gap-2">
+                        <svg className="w-5 h-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)